File: JAVA/Shirt.java
public class Shirt extends Clothing {
    private String color;
    private String sleeveType;

    public Shirt(int idProduct, String name, String brand, double price, String size, String material, String gender, String color, String sleeveType) {
        super(idProduct, name, brand, price, size, material, gender);
        this.color = color;
        this.sleeveType = sleeveType;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public String getSleeveType() {
        return sleeveType;
    }

    public void setSleeveType(String sleeveType) {
        this.sleeveType = sleeveType;
    }
}

File: CPP/Shirt.cpp
#include "Shirt.h"

Shirt::Shirt(int idProduct, std::string name, std::string brand, double price, std::string size, std::string material, std::string gender, std::string color, std::string sleeveType)
        : Clothing(idProduct, name, brand, price, size, material, gender) {
    this->color = color;
    this->sleeveType = sleeveType;
}

std::string Shirt::getColor() {
    return color;
}

void Shirt::setColor(std::string color) {
    this->color = color;
}

std::string Shirt::getSleeveType() {
    return sleeveType;
}

void Shirt::setSleeveType(std::string sleeveType) {
    this->sleeveType = sleeveType;
}
